<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMSMediaFoundation</class>
 <widget class="QDialog" name="DialogMSMediaFoundation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupColorMode">
       <property name="title">
        <string>Color Mode</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="comboColorMode">
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupROI">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Size</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="lblx0">
            <property name="text">
             <string>x0</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="RangeWidget" name="rangeX01">
            <property name="maximum">
             <number>2048</number>
            </property>
            <property name="maximumValue">
             <number>99</number>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="suffix">
             <string/>
            </property>
            <property name="tickInterval">
             <number>5</number>
            </property>
            <property name="autoSpinBoxWidth">
             <bool>true</bool>
            </property>
            <property name="tracking">
             <bool>false</bool>
            </property>
            <property name="decimals" stdset="0">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="lblx1">
            <property name="text">
             <string>x1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbly0">
            <property name="text">
             <string>y0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="RangeWidget" name="rangeY01">
            <property name="singleStep">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>2048</number>
            </property>
            <property name="maximumValue">
             <number>99</number>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="suffix">
             <string/>
            </property>
            <property name="tickInterval">
             <number>5</number>
            </property>
            <property name="autoSpinBoxWidth">
             <bool>true</bool>
            </property>
            <property name="tracking">
             <bool>false</bool>
            </property>
            <property name="decimals" stdset="0">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="lbly1">
            <property name="text">
             <string>y1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_sizeX">
            <property name="text">
             <string>x-size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinSizeX">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>666</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblsizey">
            <property name="text">
             <string>y-size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinSizeY">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>666</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnFullROI">
            <property name="text">
             <string>full ROI</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="locale">
      <locale language="German" country="Germany"/>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RangeWidget</class>
   <extends>QWidget</extends>
   <header>rangeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
